torchvision/prototype/features/_bounding_box.py

```diff
@@ -62,21 +58,15 @@ def new_like(
     def to_format(self, format: Union[str, BoundingBoxFormat]) -> BoundingBox:
         # TODO: this is useful for developing and debugging but we should remove or at least revisit this before we
         #  promote this out of the prototype state
-
```

Comment:
This is the core of what this PR is trying to resolve. The same happens for the `BoundingBox` and `SegmentationMask` feature